packages/zowe-explorer/src/dataset/DatasetTree.ts

```diff
@@ -482,8 +482,14 @@ export class DatasetTree extends ZoweTreeProvider implements IZoweTree<IZoweData
                     }
                 }
             }
-            if (profileType && this.mSessionNodes.length === 1) {
-                await this.addSingleSession(Profiles.getInstance().getDefaultProfile(profileType));
+            if (this.mSessionNodes.length === 1) {
+                try {
+                    await this.addSingleSession(Profiles.getInstance().getDefaultProfile(profileType));
+                } catch (error) {
+                    // catch and log error of no default,
+                    // if not type passed getDefaultProfile assumes zosmf
+                    this.log.debug(error);
```

Comment:
General question:
Should we avoid using debug for things that should be logged as errors ?
In this case, it might just be a warning since the user could still use the app 😋